recipes/oauthenticator/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "oauthenticator" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f174c7b2fa20028717aa10e778f66ffa896a2fbcb2ee8a78b135d5aa8a5d56ff
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
This should be changed to  `# [win]`. [According to upstream](https://github.com/jupyterhub/oauthenticator/blob/5b0b395326292d526a4c924c34c6d9d1d06818af/setup.py#L22-L24), they just don't support Windows.

Reply:
done